tests/format/handlebars/single-quote-option/attributes.hbs

```diff
@@ -1,8 +1,12 @@
 <div title='My title'></div>
 <div title="My title"></div>
 <div title='My "title"'></div>
+<div title="My 'title'"></div>
 <div title="My other 'title'"></div>
 
+<div title="{{t 'my.title'}}" />
+<div title='{{t "my.title"}}' />
+
 <a href="/{{url}}/{{url}}"></a>
 <a href='/{{url}}/{{url}}'></a>
 
```

Comment:
May we add another more complicated test case, with nested SubExpressions? (note that it passes with the current implementation)
`<div class="padding {{if foo (if fooAgain "bar" (if fooAgainAgain "bar" "foo"))}} baz" />`

Reply:
Added